tests/spread/general/bare-base/task.yaml

```diff
@@ -0,0 +1,13 @@
+summary: bare base build test
+
+execute: |
```

Comment:
there is no restore here which would cleanup everything.

Is there a general docker command we could run that would remove everything? We could run that in a more general restore (we seem to be running out of space). While I would prefer proper cleanups, we can request larger images with the "storage" key in spread.yaml, example of a different project for reference:

```yaml
    systems:
      - ubuntu-18.04-64:
          workers: 18
          image: ubuntu-1804-64
      - ubuntu-20.04-64:
          workers: 6
          image: ubuntu-2004-64
          storage: 40G
      - ubuntu-22.04-64:
          workers: 6
          image: ubuntu-2204-64
```

Reply:
we could use `docker image prune -a` but this is a bit aggressive. if these tests are run on a user's env, they'll cleanup all the images on that host.

the alternative is to cleanup per task, just the things we've created -> in this case, the .rock file + respective docker image.

wdyt?